apps/site/pages/docs/multiple-page-templates.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: 'Using specific templates in PDPs and PLPs'
+sidebar_label: 'Multiple page templates'
+---
+
+<header>
+
+# Multiple page templates
+
+</header>
+
+Multiple Page Template means that PDPs and PLPs, previously defined as singleton in the CMS, can now assume different page templates based on some rules. This approach is almost similar to the PDP and PLP, but they will be explained separately since each page type has particularities.
+
+## PDP Template
```

Comment:
In `[settings](https://vendemo.myvtex.com/admin/new-cms/faststore/pdp/edit/4bf40367-ecbf-44d4-a330-36c0d7f587b1)`, does the only possible field is `Template`?